package.json

```diff
@@ -40,6 +45,7 @@
     "@types/dotenv": "^8.2.0",
     "@types/express": "^4.0.37",
     "@types/express-session": "^1.15.3",
+    "@types/hapi__joi": "^16.0.9",
```

Comment:
What is the difference between `@types/hapi__joi` and `@types/joi`? Isn't there an overlap?

Reply:
Looks like we are not using `@types/hapi__joi`, so i have removed this